runtime/binding-kafka/src/main/java/io/aklivity/zilla/runtime/binding/kafka/internal/KafkaEventContext.java

```diff
@@ -69,10 +69,10 @@ public void apiVersionRejected(
         KafkaEventFW event = kafkaEventRW
             .wrap(eventBuffer, 0, eventBuffer.capacity())
             .apiVersionRejected(e -> e
-                .timestamp(timestamp.getAsLong())
+                .timestamp(clock.millis())
```

Comment:
We need to include the `namespacedId` here so we can know where this came from. That field should be required in the .idl so not sure why it doesn't fail here at `.build()`.